qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -21,6 +21,7 @@ void valid() {
         try {
             int aaa = this.field;
             final int twelveletter = ++aaa;
+            final int id = -1;
```

Comment:
@jrdalpra we also need negative test cases here like e.g. `it` is still not allowed